app/parse.py

```diff
@@ -8,5 +13,24 @@
     duration: str
 
 
+def get_number_from_months_text(text: str) -> str:
+    return text.split(" ")[0][:-1]
+
+
+def parse_single_course(course: Tag) -> Course:
+    name = course.select_one("a h3").text
+    description = course.find_all("p", recursive=False)[0].text
```

Comment:
The code assumes that `find_all` will always return a list with at least one element. If the HTML structure changes, this could lead to an `IndexError`. Consider adding a check to ensure the list is not empty before accessing its first element.